lib/ProductOpener/Routing.pm

```diff
@@ -408,152 +408,116 @@ sub analyze_request ($request_ref) {
 				if $log->is_debug();
 		}
 
-		if (
+		# Extract tag type / tag value pairs and store them in an array $request_ref->{tags}
```

Comment:
Could we extract this part in a separate sub (making code easier to read) ?

Reply:
Sure.